### ISSUE TITLE
arc reactor

### ISSUE THREAD

**Issue by TechDudie** [joe7575/techage#91]
have you ever seen iron man? idk if they have that in germany

in iron man theres his arc reactor, and that sounds like a good addition to ta5 (the technology doesnt exist rn)

**Comment by TechDudie** [comment]
according to some site the blue electricity flows in a loop, generating infinite electricity (which breaks the rules of physics irl)

**Comment by joe7575** [comment]
No, I don't know iron man, even it is known in Germany. But I will check, if it fits to techage

**Comment by debiankaios** [comment]
I propose also introduce stuff you need using fusion reactor.
